Example/Tests/Configuration.swift

```diff
@@ -24,7 +24,15 @@ class SchibstedAccountConfiguration: QuickConfiguration {
 
             Utils.cleanupKeychain()
             PasswordlessTokenStore.clear()
-            Networking.proxy = TestingNetworkingProxy()
+
+            //
+            // Before each test, make sure we reset the proxy to use the testing networking proxy incase a test changes the configuration
+            // And make sure the internal forwarding proxy if the testing proxy is the default one.
+            //
+            let testingNetworkingProxy = TestingNetworkingProxy()
+            testingNetworkingProxy.internalProxy = DefaultNetworkingProxy()
```

Comment:
Should this be `StubbedNetworkingProxy`? We changed that earlier to make the stubbing work.